src/lib/server/storages/health.ts

```
@@ -0,0 +1,121 @@
+ * @param storage The storage object to check for desynced backups.
+ * @return A ResultAsync containing an array of desynced backups or an error message.
+ */
+export async function getPruneDesyncBackups(storage: typeof storages.$inferSelect): Promise<ResultAsync<Desyncedbackup[], string>> {
+    const backups = getUnprunedSnapshotByStorageId(storage.id);
+    if (backups.length === 0) {
+        return ok([]);
+    }
+
+    const res = await getRepositorySnapshots(
+        storage.url,
+        storage.password!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)